pf4j/src/main/java/org/pf4j/processor/ExtensionAnnotationProcessor.java

```diff
@@ -130,10 +130,10 @@ public boolean process(Set<? extends TypeElement> annotations, RoundEnvironment
             }
         }
 
-        // write extensions
-        if (extensions.size() > 0) {
-            storage.write(extensions);
-        }
+        // even invoke if no extensions were found: an empty extensions descriptor
+        // is semantically correct and prevents classloaders falling back to descriptors
+        // from dependendencies
```

Comment:
Please fix the typo in `dependendencies`.

Reply:
Done with a bit of rewording to (hopefully) improve documentation of what's going on.